src/http/radio-http.js

```diff
@@ -4,12 +4,22 @@ import {assert} from '../validation';
 
 const getStations = createJsonRoute(function(req, res) {
   const params = assert({
-    radioId: req.query.radioId,
-    cityId: req.query.cityId,
-    cityName: req.query.cityName,
+    id: req.params.id,
+    city: req.query.city,
   }, 'radioParams');
 
-  return radioCore.getStations(params);
+  return radioCore.getStations(params).then(results => {
+    if (req.params.id !== undefined) {
+      if (results.length > 1) {
+        throw new Error('Unexpected number of rows');
+      }
+      // Respond with a single object if queried by id.
+      return results[0];
```

Comment:
Oh this seems to be it's own route. Then we should throw 404 if object is not found.